[DA] QA 2차 반영

kimkyunghun3 opened the pull request:
# 개요
- 🔗  이슈링크 : #139 

## 작업 내용
- ToastView 실패 시 description 제대로 나오는 것 확인(서버 수정)
- 응모 후 디테일 재진입 시 응모버튼 응모완료 되도록 로직 변경
- 카테고리 명칭 한글화(서버 수정)
- 카운트 다운 초 칼라 적용
- scrollIndicator 제거
- 토스트 뷰 호출 후 scroll 일시적 멈춤 현상 해결

### 스크린샷

<img src="https://user-images.githubusercontent.com/52434820/184782585-4c298f46-dda1-4427-9aaf-4fe1fee6085f.gif" width="400">

kimkyunghun3 committed:
:recycle: Timer 초 칼라 변경

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 committed:
:recycle: 응모화면 scrollIndicator 제거

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 committed:
:recycle: ToastView animation후 scroll 안되는 이슈 해결

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 committed:
:recycle: 응모 완료된 기프티콘 응모 디테일 진입시 응모완료되도록 변경

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 committed:
:recycle: 카테고리 한글 변경에 따른 수정

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 committed:
:recycle: 현재 유효시간 안나오는 에러 수정

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

szzang-kr reviewed (changes requested):
Toast의 사라지는 애니메이션 적용했으니까 

 ToastView.swift의  `func configureToastView()`안에 있는 
```swift
        self.wrapperView.removeFromSuperview()
        self.dimView.removeFromSuperview()
```

내용을 삭제해야할것 같아요!

kimkyunghun3 committed:
:recycle: 불필요한 코드 삭제

# :tada: = 프로젝트 시작 (#)
# :sparkles: = create 파일 생성 시 (#)
# :bulb: = update 코드 및 UI 업데이트, 기능 구현 시 (#)
# :rocket: = release 배포 시 (#)
# :recycle: = refactor 리팩토링 시 (기능은 같은데 코드가 달라지는경우) (#)
# :memo: = 문서 작성 시 (#)
# :art: = 디자인 시스템 관련 내용 (Color / Asset) (#)
# :hammer: = 테스트 코드 작성 (#)
################

# 커밋 타입 : 커밋 제목 (계정명/저장소명#이슈번호) 형식으로 작성하며 제목은 최대 50글자 정도로만 입력
# 제목을 아랫줄에 작성, 제목 끝에 마침표 금지, 무엇을 했는지 명확하게 작성

kimkyunghun3 commented:
> Toast의 사라지는 애니메이션 적용했으니까
> 
> ToastView.swift의 `func configureToastView()`안에 있는
> 
> ```swift
>         self.wrapperView.removeFromSuperview()
>         self.dimView.removeFromSuperview()
> ```
> 
> 내용을 삭제해야할것 같아요!

아하 그러네요! 삭제했습니다!
